src/styles/main.scss

```diff
@@ -0,0 +1,97 @@
+@import './utils/variables';
+
+$padding: 10px;
+$gap: 1px;
+$total-gap: $gap * 8;
+$square-side: 100px;
+$line-width: ($square-side * 7) + ($padding * 2) + $total-gap;
+$day-of-the-week: (
+  'mon': 0,
+  'tue': (
+    ($square-side + $gap),
+  ),
+  'wed': (
+    ($square-side + $gap) * 2,
+  ),
+  'thu': (
+    ($square-side + $gap) * 3,
+  ),
+  'fri': (
+    ($square-side + $gap) * 4,
+  ),
+  'sat': (
+    ($square-side + $gap) * 5,
+  ),
+  'sun': (
+    ($square-side + $gap) * 6,
+  ),
+);
+
+body {
+  margin: 0;
+}
+
+.wrapper {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  box-sizing: border-box;
+  padding: $padding;
+  width: $line-width;
+
+  transform: translateX(1px);
```

Comment:
why you added this style?, you don't need it :)

Reply:
I was sure that this task would have better results, truly)
But still, thank you for closely watching!)

I won't pass any tests without this transition(

Reply:
@mrkovaliv suddenly I realized that I missed asking you about my test failure)
so one thing is that I had to transform the whole calendar to pass 3 tests.
and another is about the flex gap, I suppose, please see the attached photo. 

It looks like every next section increased the difference from the refer
![photo_2024-03-08_11-30-33](https://github.com/mate-academy/layout_calendar/assets/102412814/ee507327-3caf-4a0d-b284-9cb44d53ad99)
ence because of the flex gap. 

Should I worried so about tests?

Reply:
no, everything works fine 👍